BAZProject/TrendingModule/MainView/TrendingTableViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  TrendingTableViewCell.swift
+//  BAZProject
+//
+//  Created by avirgilio on 30/01/23.
+//
+
+import UIKit
+
+class TrendingTableViewCell: UITableViewCell {
+    
+    let manageImgs = LoadRemotedata()
+    
+    @IBOutlet weak var imgImageMovie: UIImageView!
+    
+    @IBOutlet weak var lblMovieTitle: UILabel!{
+        didSet{
+            lblMovieTitle.numberOfLines = 2
+            lblMovieTitle.font = UIFont(name: "ArialMT", size: 20.0)
+        }
+    }
+    
+    @IBOutlet weak var lblMovieRate: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    func loadMoviesInfo(arrayMovies: [ResultMovie], index: IndexPath){
```

Comment:
SwiftLint fix:
```suggestion
    func loadMoviesInfo(arrayMovies: [ResultMovie], index: IndexPath) {
```
Si te agrada dejar un espacio entre la declaración de tu función y su contenido está bien, pero que sea así en todas tus funciones, para ser congruente con tu estilo de escritura. Caso contrario, no dejes el espacio.